src/read_next/read_next.jsx

```diff
@@ -1,8 +1,11 @@
-import React, { useEffect, useState } from "react";
-import "./read_next.css";
+import React, { useEffect, useState } from 'react';
+import Pagination from "https://cdn.skypack.dev/rc-pagination@3.1.15";
+import './read_next.css';
 import { Link, useParams } from "react-router-dom";
-
 const ReadNext = () => {
+    const [perPage, setPerPage] = useState(3);
+    const [size, setSize] = useState(perPage);
+    const [current, setCurrent] = useState(1);
```

Comment:
very difficult to understand what these variables mean. choose more informative names